linux/ci-build.sh

```diff
@@ -1,5 +1,12 @@
-make -j8 -f Makefile.tests || exit
-cp -R ../resources ./OpenSupaplex-linux
+if [[ "$(uname)" != Linux ]]; then
+	# this is required only for macOS
+	export CFLAGS="-I/usr/local/include" LDFLAGS="-L/usr/local/lib"
+fi
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release || exit
+make -j8 opensupaplex-test || exit
+cp -R ../../resources ./OpenSupaplex-linux
 rm -rf ./OpenSupaplex-linux/audio-lq # we don't need low quality audio here
 rm -rf ./OpenSupaplex-linux/audio-mq # we don't need medium quality audio here
-cp opensupaplex ./OpenSupaplex-linux/OpenSupaplex
+cp opensupaplex-test ./OpenSupaplex-linux/OpenSupaplex
```

Comment:
Same here about tests and macOS: feel free to get rid of all that. Tests will be handled in `tests`, and whether they run on macOS or Linux should be irrelevant to these scripts.

Reply:
If I just switch to cmake, but don't touch this, I'd essentially break the test. Do you prefer merging in a way which breaks test?

Reply:
That won't happen 😄 I just moved them to `tests`: da367ba8b5808f4019ed1b4ec441fc66654798e0

Now you can get rid of everything test and macOS related in this PR 😄